components/LevitatingLink.js

```diff
@@ -1,7 +1,8 @@
-export default function LevitatingLink ({ href, children }) {
+export default function LevitatingLink ({ href, children, active }) {
```

Comment:
A LevitatingLink should not take an `active` prop, because the `active` prop is for the navigation. A general UI component like LevitatingLink does not need to know it.

You can add an `active` class outside this component, e.g., inside `NavigationLink`.